juniper_official/routing/collect-sr-lsp-stats.rule

```diff
@@ -76,25 +76,25 @@ healthbot {
                             products MX {
                                 sensors jnpr-sr-lsp;
                                 platforms MX960 {
-                                    releases 22.1R1 {
```

Comment:
Does this change mean that 22.1R1 is not supported on these platforms?
If yes, does PF takes care of not instantiating the rules?